README.md

```diff
@@ -13,11 +13,11 @@ cross-platform, high-performance, secure, and reliable. The libraries are bound
 to JS by the [awscrt](https://github.com/awslabs/aws-crt-nodejs) package.
 
 *__Jump To:__*
-* [Installation](#Installation)
+* [Installation](#installation)
 * [Samples](samples)
-* [Getting Help](#Getting-Help)
+* [Getting Help](#getting-Help)
```

Comment:
I think all letters in the anchor must be lowercase.
I couldn't find this rule in the github docs, but that's how github itself generates these anchors (the link to the left of each header). Also, other markdown docs state that anchors should be all lowercase.

Reply:
Yes, thanks for pointing this out